package.json

```diff
@@ -17,8 +30,10 @@
   "homepage": "https://github.com/auth0/jwt-decode#readme",
   "scripts": {
     "dev": "rollup --sourcemap --config --configPlugin typescript",
-    "build": "rimraf build && rollup --sourcemap --config --configPlugin typescript --environment NODE_ENV:production",
-    "postbuild": "echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
```

Comment:
Do we no longer need this?

Reply:
Yeah, you are right. Now that we're using the `.cjs` extension this should work without this. I'll remove it.